advanced/list-of-log-names.md

```diff
@@ -1,9 +1,8 @@
----
-description: On this page all the available logs are listed
----
+# List Of Log Names
 
+---
 
-# [List Of Log Names](../advanced/list-of-log-names.md#ListOfLogNames)
+description: On this page all the available logs are listed
```

Comment:
No, this desc goes on top between the `---` things like before, likely some GitBook specific meaning